beluga_tutorial/README.md

```diff
@@ -0,0 +1,505 @@
+# Simple MCL Implementation Using Beluga
```

Comment:
Or, alternatively, we could have this tutorial be the continuation of the key concepts section on MCL. If that was the case, after text de-duplication, it would be fine IMO to share the conceptual discussion between both articles (that one and this one).

Reply:
From my perspective, it is essential to define some key concepts and formulas or mathematical notation at the beginning of the tutorial, as I use them to explain how the filter works step by step. Therefore, it seems appropriate to position this tutorial as a continuation of the key concepts section and erase the explanation of the concepts from here. Additionally, I want to include the relevant formulas at the start of the tutorial to avoid the need to refer back to the concepts page repeatedly.

Reply:
That sounds reasonable to me.